### PR TITLE
Remove czip/unzip references

### PR DESCRIPTION
**Pull request by GravisZro**
Removes references to the czip/unzip libraries which are to be replaced with zlib.

## Pull Request Type

- [ ] GitHub Workflow changes
- [ ] Documentation or Wiki changes
- [x] Build and Dependency changes
- [ ] Runtime changes
  - [ ] Render changes
  - [ ] Audio changes
  - [ ] Input changes
  - [x] Network changes
  - [ ] Other changes

### Checklist

- [ ] I have tested my changes locally and verified that they work as intended.
- [ ] I have documented any new or modified functionality.
- [ ] I have reviewed the changes to ensure they do not introduce any unnecessary complexity or duplicate code.
- [ ] I understand that by submitting this pull request, I am agreeing to license my contributions under the project's license.

### CODE REVIEW

File: Descent3/mission_download.cpp [hunk 1]
```diff
@@ -694,7 +693,7 @@ void _get_zipfilename(char *output, char *directory, char *zipfilename) {
 // return 0 on failure
 // return 1 on success
 int msn_ExtractZipFile(char *zipfilename, char *mn3name) {
-
+/*
```
Comment (Lgt2x) [inline Descent3/mission_download.cpp:696]: so you just disable the feature ?

Comment (GravisZro) [reply Descent3/mission_download.cpp:696]: Most of "mission download" was already commented out due to lack of a location to download from. The only thing I did was comment out the unzipping after it was downloaded. This seems to be the only place that the deflate decompression algorithm is used.

Comment (JeodC) [reply Descent3/mission_download.cpp:696]: Mission files hosted by a server contain a url pointing clients to the download location. If zlib is meant to replace, it should be integrated here.

Comment (GravisZro) [reply Descent3/mission_download.cpp:696]: Presently, downloading is entirely disabled. Replacing the compression code right now is a moot point and cannot be tested.

Comment (Lgt2x) [reply Descent3/mission_download.cpp:696]: This change will make our lives harder when/if we want to enable mission download again, I don't think we should merge it as is

**Pull request closed without merging**